install/install-mac-deps.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+echo -e "\n\nInstalling homebrew, and MacOS dependencies using the same\n"
+
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+echo 'export PATH="/usr/local/opt/gettext/bin:$PATH"' >> $HOME/.bash_profile
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expressions don't expand in single quotes, use double quotes for that.](https://app.codacy.com/manual/akotlar/bystro/pullRequest?prid=4887428)